src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -241,6 +241,9 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration,
       case ActivationNotification(recipentUserId) =>
         thurloe.service.Notification(Option(recipentUserId), None, None, templateId, Map.empty, Map.empty, Map.empty)
 
+      case AzurePreviewActivationNotification(recipentUserId) =>
+        thurloe.service.Notification(Option(recipentUserId), None, None, templateId, Map.empty, Map.empty, Map.empty)
+
```

Comment:
I'm going to let Orch differentiate between the `ActivationNotification` and `AzurePreviewActivationNotification` cases. Because we're using PubSub to communicate between Orch and Thurloe, we can't do any sort of proxy header/token inspection to determine what type of user we're dealing with. Right now, Orch is in the best position to make that differentiation.

There are better ways to do this, but it will take more engineering time than we have right now. I think the ideal case is to:

1) Give Thurloe the ability to differentiate between cloud providers (could even go further and look for things like which branding of the site is being used- Anvil, Rare-X, etc)
2) Upgrade SendGrid API in Thurloe to latest version, which supports dynamic templating
3) Convert all of our existing legacy templates to dynamic templates in SendGrid
4) Pass some variable into the dynamic template, i.e. `cloudProvider: oneOf[GCP, Azure]` and dynamically generate the email content based on that. Then we can simplify a few areas in the Thurloe and Orch code.

Moving to dynamic templates makes a lot of other things easier too and allows us to be more flexible with our email notification.

Reply:
I see - thanks for the explanation. Having Orch differentiate sounds reasonable for now. Happy to take a look at the Orch PR too